### PR TITLE
WireframeBranch

### PR DESCRIPTION
**Pull request by kayjeee**
set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I developed a first working version of the app following a wireframe, but without styling - just focus on functionality. In following activities, I will consume the Leaderboard API using JavaScript async and await and add some styling.

General requirements
Made sure that there are no linter errors.
Made sure that you used correct Gitflow.
Made sure that you documented your work in a professional way.
Followed our list of best practices for JavaScript.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  // Get input values
+  const input1Value = document.getElementById('input1').value;
+  const input2Value = document.getElementById('input2').value;
+
+  // Perform form submission logic here
+  console.log('Input 1:', input1Value);
+  console.log('Input 2:', input2Value);
```
Comment (AmaduKamara) [inline src/index.js]: - [ ] Let's try to not commit console logs in our codebase. It's a good convention to not push any console.log() code in your commits. Kindly remove them 👍🏽

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                    <label for="input1">Input 1:</label>
+                    <input type="text" id="input1" name="input1" required>
+                    <label for="input2">Input 2:</label>
+                    <input type="password" id="input2" name="input2" required>
+                    <input type="submit" value="Submit" class="submit-button">
+                
+                </form>
+            </div>
+        </section>
+    </div>
```
Comment (AmaduKamara) [inline src/index.html]: - [x] Kindly match your app structure with the original wireframe. Currently, you are missing some placeholders in your design that do not look like the wireframe. Kindly see the screenshot below. 👍🏽
- [x] Kindly have all placeholders in the wireframe present in your app structure. 👍🏽

**Your Structure**

Screenshot 2023-05-24 at 5 26 59 PM


**The Wireframe Structure**

Screenshot 2023-05-24 at 5 25 44 PM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️ 

Hi @kayjeee,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅  Used Webpack correctly 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.html (hunk 2 above)
Comment (kayjeee) [reply src/index.html]: made required changes ,please look again

File: src/index.js (hunk 1 above)
Comment (roshan-bajgain) [inline src/index.js]: - [x] As requested by the previous reviewer please remove this `console.log` as it is considered to be bad practice.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @kayjeee, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Good job on implementing some changes ✔️ 
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (kayjeee) [reply src/index.js]: okay,thank you,removed it

**Review by abdallahmalima — APPROVED** [review]
Hi @kayjeee , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Use webpack ✔
- Basic project structure✔
- Design resembles wireframe✔
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**